src/main/java/chess/Board.java

```diff
@@ -0,0 +1,87 @@
+package chess;
+
+import chess.pieces.Pawn;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Board {
+    private static final int SIZE = 8;
+    private static final int WHITE_PAWNS_LINE = 1;
+    private static final int BLACK_PAWNS_LINE = 6;
+
+    private final List<List<Pawn>> pawns;
+
+    public Board() {
+        this.pawns = new ArrayList<>();
+    }
+
+    public void initialize() {
+        for (int line=0; line<SIZE; line++) {
+            pawns.add(line, getInitialPawns(line));
+        }
+    }
+
+    public void print() {
+        StringBuilder stringBuilder = new StringBuilder();
+        pawns.stream().sorted((list1, list2) -> -1)
```

Comment:
```suggestion
        pawns.stream().sorted((e1, e2) -> -1)
```
sorted 함수에 들어가는 comparator 의 매개변수는 리스트가 아니라 아이템이기 때문에 o1, o2 또는 여기서는 p1, p2 등을 사용해야 합니다. 

그리고 단순히 역순으로 출력하기 위해서 sorted를 사용하는 건 성능과 읽는 사람의 가독성 측면에서 좋지 않을 거 같아요. 왜 정렬했지? 하고 몇 초간 고민하게 되거든요. 
주석으로 //리스트를 뒤집어 출력 정도만 달았어도 좋았을 거 같구요.

가능한 다른 방법을 고민해 보는 것도 좋을 것 같아요.

Reply:
가능한 다른 방법이 있는지 고민해 보겠습니다. 감사합니다!